vllm/worker/model_runner.py

```diff
@@ -68,6 +70,9 @@ def __init__(
         # cache in_wsl result
         self.in_wsl = in_wsl()
 
+        self.kv_cache_dtype = self.cache_config.quant_method if (
+            self.cache_config and self.cache_config.quant_method) else "auto"
```

Comment:
why is the `kv_cache_dtype` equals to `quant_method`?

Reply:
Sure. It is unclear here. Replace `quant_method` with `cache_dtype_str`.